joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/LambdaTests.scala

```diff
@@ -126,14 +119,14 @@ class LambdaTests extends JavaSrcCode2CpgFixture {
           val fallbackLocal = cpg.method.name(".*lambda.*").local.name("fallback").head
           fallbackClosureBinding.closureBindingId shouldBe fallbackLocal.closureBindingId
 
-          fallbackClosureBinding.outE.collectAll[Ref].map(_.inNode()).l match {
+          fallbackClosureBinding.outE.asScala.collectAll[Ref].map(_.inNode()).l match {
```

Comment:
Can we introduce `collectAll` on `outE` and friends?

Reply:
How about an extension method `collectAll` on `java.util.Iterator`? I guess that's what fabs had in mind here.